uwblueprint/ccbc
# Dhruv/set password admin

## Notion ticket link

[Admin Authentication: Setup Password email](https://www.notion.so/uwblueprintexecs/Task-Board-db95cd7b93f245f78ee85e3a8a6a316d)



## Implementation description
* added 2 new components on front end 
* modified /register route to set password instead of sending verifying email 
* added functions to authService and AuthApiClient.ts



## Steps to test
1. First use the signup page to create an account (make sure that your browser's local storage is clear)
2. Go to the email you used and click on the link that was in the mail
3. set your new password (make sure that your browser's local storage is clear before)
4. go to localhost/default to logout 
5. signin with your new password 



## What should reviewers focus on?
* Added a route to AuthActions 
* Added a SetPassword and AuthActions components on the front end
* Modified /register endpoint to send setpassword email instead of verifyemaill link


## Checklist
